driver-sync/src/test/functional/com/mongodb/client/OcspTest.java

```diff
@@ -0,0 +1,58 @@
+package com.mongodb.client;
+
+import com.mongodb.MongoException;
+import com.mongodb.MongoNamespace;
+import com.mongodb.client.test.CollectionHelper;
+import org.bson.BsonDocument;
+import org.bson.BsonInt32;
+import org.bson.Document;
+import org.bson.codecs.DocumentCodec;
+import org.junit.Before;
+import org.junit.Test;
+
+import static com.mongodb.ClusterFixture.serverVersionAtLeast;
+import static com.mongodb.client.Fixture.getDefaultDatabaseName;
+import static org.junit.Assert.fail;
+import static org.junit.Assume.assumeTrue;
+
+public class OcspTest {
+    private static final String COLLECTION_NAME = "test";
+    private final boolean shouldSucceed = Integer.parseInt(System.getenv("OCSP_TLS_SHOULD_SUCCEED")) == 1;
```

Comment:
Can this variable be moved to build.gradle - it'll help keep setting / checking of system variables in one place.

Reply:
This variable is set on a per-test basis in `.evg.yml` and passed to `run-ocsp-test.sh`.

Reply:
Can it still be set as a gradle property and passed in the conventional way via Gradle extra vars? eg: `-Porg.mongodb.test.ocsp.tls.should.succeed=${OCSP_TLS_SHOULD_SUCCEED}`.

It just keeps the configuration standard.  You could even skip the test if its not set.

Reply:
The Gradle property has been added, and I have added the method `ClusterFixture.getOcspShouldSucceed` to get the success value.